docs/concepts/adapters.md

```diff
@@ -134,3 +139,27 @@ async def _request_instrument(self, request: RequestInstrument) -> None:
 
     self._handle_instrument(instrument, request.id, request.params)
 ```
+
+The `DataEngine` which is an important component in Nautilus links a request with a `DataClient`.
+For example a simplified version of handling an instrument request is:
+
+```python
+# nautilus_trader/data/engine.pyx
+
+self._msgbus.register(endpoint="DataEngine.request", handler=self.request)
```

Comment:
We should probably point out somewhere that this comes for free with the base classes, so it looks less complex.